packages/sdks/snippets/react-sdk-next-pages/src/components/NavBar.tsx

```diff
@@ -0,0 +1,17 @@
+import type { BuilderContent } from '@builder.io/sdk-react';
+import React from 'react';
+
+export function NavBar({ links }: { links: BuilderContent | null }) {
```

Comment:
you're typing links as `BuilderContent | null` but it will only get passed to this component when links is defined because of `{links && <NavBar links={links} />}`